site/gatsby-site/gatsby-browser.js

```diff
@@ -1,11 +1,11 @@
 export const onServiceWorkerUpdateReady = () => {
   const answer = window.confirm(
-    `This website has been updated. ` +
-      `Reload to display the latest version?`
-  )
+    `This website has been updated. ` + `Reload to display the latest version?`
```

Comment:
It looks like linting was not quite aggressive enough here. No need to change this, but noting it should likely just be one string.